ehr2vec/effect_estimation/estimate.py

```
@@ -0,0 +1,99 @@
+        logit_perturb = full_lq + eps * h
+        return expit(logit_perturb), h
+
+    def run_tmle_binary(self):
+        """
+        This is for CV-TMLE on binary outcomes yielding risk ratio with 95% CI. Read Levi et al for methodological details.
+        Influence curves coded from Gruber S, van der Laan, MJ. (2011).
+
+        """
+
+        print("running CV-TMLE for binary outcomes...")
+        q_t0, q_t1, g, t, y, truncatel = (
```

Ruff (0.8.2)

Local variable `truncatel` is assigned to but never used

Remove assignment to unused variable `truncatel`

(F841)